Tests/NIOTests/ChannelTests.swift

```diff
@@ -157,9 +157,9 @@ public class ChannelTests: XCTestCase {
         for _ in 0..<bufferSize {
             buffer.write(staticString: "a")
         }
-        
 
-        let lotsOfData = Int(Int32.max)
+
+        let lotsOfData = Int(Int32.max) - bufferSize
```

Comment:
Why now it’s only 2 MB? It’s 2gb - 2mb, no? It was a problem that ‘var written’ is Int32, and on line 166 it crash because of number overflow. It’s seems that I need to set 64bit explicitly 😅

Reply:
Yep, looks good 👍🏼